ocpmodels/trainers/forces_trainer.py

```diff
@@ -358,13 +365,15 @@ def predict(
 
         if results_file is not None:
             print(f"Writing results to {results_file}")
-            # EvalAI expects a list of dicts with energy and forces
+            # EvalAI expects a list of dicts with ids, energy, and forces
             evalAI_results = []
-            for energy, forces in zip(
-                predictions["energy"], predictions["forces"]
+            for rid, energy, forces in zip(
+                predictions["ids"],
+                predictions["energy"],
+                predictions["forces"],
             ):
                 evalAI_results.append(
-                    {"energy": energy, "forces": forces.tolist()}
+                    {"id": rid, "energy": energy, "forces": forces.tolist()}
```

Comment:
Maybe I missed it but these force predictions should just be on free atoms, right?

Reply:
Depends how we deal with it on the server side, if we're giving it all forces and extracting free or just free.